### PR TITLE
`Development`: Document the iOS app's communication features

### PR DESCRIPTION
**Pull request by nityanandaz**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

Document the iOS app's communication feature and App Store release.

### Description

https://live.rbg.tum.de/w/artemisintro/47329

replaces

https://live.rbg.tum.de/w/artemisintro/35219

#### Code Review
- [x] Code Review 1
- [x] Code Review 2

## Summary by CodeRabbit

- **Documentation**
  - Updated availability information for the iOS application, now accessible through the App Store and TestFlight.
  - Added clarifications on the differences between mobile and web applications.
  - Introduced instructions for restoring cached message drafts on iOS.

- **Bug Fixes**
  - Adjusted conditional classes in the clone repository button to better handle SSH and version control settings.

### CODE REVIEW

File: docs/user/mobile-applications.rst [hunk 1]
```diff
@@ -50,18 +50,22 @@ The Android application supports the following features:
 Installation
 ^^^^^^^^^^^^
 
-The Android application is currently only available in beta state and can be downloaded over `Firebase App Distribution <https://appdistribution.firebase.dev/i/f5dedbb0fc6dc0da>`_. Make sure to follow the steps described by Firebase App Distribution to install the application.
+The Android application is currently only available in beta state and can be downloaded over `Firebase App Distribution <https://appdistribution.firebase.dev/i/f5dedbb0fc6dc0da>`_.
```
Comment (marlon-luca-bu) [inline docs/user/mobile-applications.rst:53]: Shouldn't that be iOS?

File: docs/user/mobile-applications.rst [hunk 2]
```diff
@@ -138,6 +140,7 @@ The iOS application supports the following features:
     #. Add emoji reactions
     #. Create new conversations
     #. Edit conversation settings
+    #. Restore cached messages drafts for up to one day
```
[the remark this answers is not in the document]
Comment (nityanandaz) [reply docs/user/mobile-applications.rst:143]: The video explains the context

File: src/main/webapp/app/shared/components/clone-repo-button/clone-repo-button.component.html [hunk 3]
```diff
@@ -44,7 +44,7 @@ <h5>{{ cloneHeadline | artemisTranslate }}</h5>
                 class="clone-url"
                 [ngClass]="{
                     'url-box-remove-line-left': sshEnabled && ((useSsh && !localVCEnabled) || !useSsh),
-                    'url-box-remove-line-right': !localVCEnabled && !(sshEnabled && ((useSsh && !localVCEnabled) || !useSsh)),
+                    'url-box-remove-line-right': !localVCEnabled && !(sshEnabled && ((useSsh && !localVCEnabled) || !useSsh))
```
Comment (nityanandaz) [inline src/main/webapp/app/shared/components/clone-repo-button/clone-repo-button.component.html:47]: Due to the `husky` pre commit hook

File: docs/user/mobile-applications.rst (hunk 1 above)
Comment (nityanandaz) [reply docs/user/mobile-applications.rst]: Formatting the document got in the way :)

File: docs/user/mobile-applications.rst [hunk 4]
```diff
@@ -159,7 +159,8 @@ The iOS application supports the following features:
 Installation
 ^^^^^^^^^^^^
 
-The iOS application is currently only available in beta state and can be downloaded over `TestFlight <https://testflight.apple.com/join/WTwsKbjr>`_. Make sure to follow the steps described by TestFlight to install the application.
+The iOS application is available in the `App Store <https://apps.apple.com/de/app/artemis-learning/id6478965616>`_ and as a beta via `TestFlight <https://testflight.apple.com/join/WTwsKbjr>`_.
+Make sure to follow the steps described by TestFlight to install the application.
```
[the remark this answers is not in the document]
Comment (nityanandaz) [reply docs/user/mobile-applications.rst]: https://live.rbg.tum.de/w/artemisintro/43388?video_only=1&t=0 is not broken

File: docs/user/mobile-applications.rst [hunk 5]
```diff
@@ -145,6 +147,10 @@ The iOS application supports the following features:
         Video tutorial of the Messaging features on TUM-Live.
     </iframe>
 
+.. note::
```
[the remark this answers is not in the document]
Comment (nityanandaz) [reply docs/user/mobile-applications.rst]: Outdated

File: src/main/webapp/app/shared/components/clone-repo-button/clone-repo-button.component.html [hunk 6]
```diff
@@ -44,7 +44,7 @@
...
```
[the remark this answers is not in the document]
Comment (nityanandaz) [reply src/main/webapp/app/shared/components/clone-repo-button/clone-repo-button.component.html]: 044127e34e0541f3e6725bd60234e3845cce2424

File: docs/user/mobile-applications.rst [hunk 7]
```diff
@@ -138,10 +140,11 @@ The iOS application supports the following features:
     #. Add emoji reactions
     #. Create new conversations
     #. Edit conversation settings
+    #. Restore cached messages drafts for up to one day
```
[the remark this answers is not in the document]
Comment (nityanandaz) [reply docs/user/mobile-applications.rst]: 044127e34e0541f3e6725bd60234e3845cce2424

File: docs/user/mobile-applications.rst [hunk 8]
```diff
@@ -159,7 +162,8 @@
 Installation
 ^^^^^^^^^^^^
 
-The iOS application is currently only available in beta state and can be downloaded over `TestFlight <https://testflight.apple.com/join/WTwsKbjr>`_. Make sure to follow the steps described by TestFlight to install the application.
+The iOS application is available in the `App Store <https://apps.apple.com/de/app/artemis-learning/id6478965616>`_ and as a beta via `TestFlight <https://testflight.apple.com/join/canwoVCk>`_.
+Make sure to follow the steps described by TestFlight to install the application.
```
[the remark this answers is not in the document]
Comment (nityanandaz) [reply docs/user/mobile-applications.rst]: 044127e34e0541f3e6725bd60234e3845cce2424

File: src/main/webapp/app/shared/components/clone-repo-button/clone-repo-button.component.html (hunk 3 above)
Comment (nityanandaz) [reply src/main/webapp/app/shared/components/clone-repo-button/clone-repo-button.component.html]: 044127e34e0541f3e6725bd60234e3845cce2424

**Review by anian03 — COMMENTED** [review]
Looks good in general – although the TestFlight link does not seem to work I believe. It tells me "This beta app does not accept new testers". Is this intentional?

**Comment by nityanandaz** [comment]
> Looks good in general – although the TestFlight link does not seem to work I believe. It tells me "This beta app does not accept new testers". Is this intentional?

I set it to 32 in App Store Connect. It was indeed 0 :)

**Review by anian03 — APPROVED** [review]
Thanks for the fix – looks good to me now :)

**Review by marlon-luca-bu — APPROVED** [review]
Looks good to me 👍

**Review by SimonEntholzer — APPROVED** [review]
lgtm

**Pull request merged**